src/Components/Artist/CollectionsRail/CollectionsRail.tsx

```diff
@@ -0,0 +1,44 @@
+import { Box, Sans, Spacer } from "@artsy/palette"
+import { ArrowButton, Carousel } from "Components/v2"
+import React, { SFC } from "react"
+import styled from "styled-components"
+import { ArtistCollectionEntity } from "./CollectionEntity"
+
+interface ArtistCollectionsRailProps {
+  collections?: any
```

Comment:
Is there a `Collection` typescript type we can use here?

Reply:
This is an incremental PR, but we will have a typing soon -- we are in the process of adding new fields to kaws and the schema has not yet been defined. This will be updated as soon as the API supports data to fetch with relay.